nb/pd-pandas-intro.ipynb

```diff
@@ -2305,31 +1055,33 @@
    "source": [
     "Everything got converted to Python objects!\n",
     "That is the caveat of filling NaNs between columns, i.e. we lose data types.\n",
-    "We can fix this with some data munging.\n",
+    "We can easily fix this now."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_fixed = data_full.convert_dtypes()\n",
```

Comment:
Reminder to myself: check pandas version for this functionality. I don't remember it :)